README.md

```diff
@@ -16,5 +17,60 @@ For now, repository contains:
   2. can serialize your AST to and from s-expressions, either to look at it, or for interchange.
 
 Outside of those notable classes and some utilities, there are 2 important subfolders:
+
 - `sexpr`: facilities for reading and writing S-Expressions. Mostly for use by Wellformed, but tested on a necessary-seeming subset of the original proposed RFC.
 - `tla`: under-construction parser for the TLA+ modeling language, which has been invaluable in stress-testing all the generic machinery. The test suite involves trying to parse the full TLA+ Examples folder.
+
+# Getting Started
```

Comment:
This might be better higher up in the file? I'm not sure; if the text above was any longer, this should definitely be higher.

Reply:
Maybe "developer guide" as title?

Realizing that someone who wants to use DCal as a user will need a whole other set of docs, and that can come later (I don't even know the process yet).